index.bs

```diff
@@ -3650,8 +3650,8 @@ Note: This means attestation statement format identifiers based on domain names
 Implementations MUST match WebAuthn attestation statement format identifiers in a case-sensitive fashion.
 
 Attestation statement formats that may exist in multiple versions SHOULD include a version in their identifier. In effect,
-different versions are thus treated as different formats, e.g., `packed2` as a new version of the `packed` attestation statement
-format.
+different versions are thus treated as different formats, e.g., `packed2` as a new version of the `packed` [=attestation statement
```

Comment:
nit: I'd link directly to #packed-attestation section here, rather than the definition of `attestation statement format`.